website/docs/secrets/managing-secrets-with-sops.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: Managing secrets with Sops
+hide_title: true
+---
+
+# Managing secrets with Sops
+
+
+In this document, we will cover the prerequisites for using [SOPS](https://github.com/mozilla/sops) with Weave GitOps Enterprise, and how to configure SOPS for your existing Kubernetes cluster to work with GPG and age keys.
+
+For a more advanced setup for SOPS with flux, please refer to this [guide](https://fluxcd.io/flux/guides/mozilla-sops/). 
+
+
+## Encrypting secrets using GPG/OpenPGP
```

Comment:
Did you consider adding in this doc just the bits relevant for Wego, and assuming that key pair exists (for example) and reference those guides (sops and flux documentation)?

Reply:
I already had this discussion with Sarah and Mostafa 
we found that it's easier for the user to read all the steps in one place specially flux documentation mentions the encryption steps in the same section of the key generation and also it recommends to delete the generated secrets which are needed in this document to export the public key as a kubernetes secret